packages/main/.eslintignore

```diff
@@ -8,3 +8,4 @@ rollup.config*.js
 wdio.conf.js
 postcss.config.js
 package-scripts.js
+thirdparty
```

Comment:
For `.eslintignore` we don't provide a central file, each library must copy/paste it basically. Therefore this will only have effect for `main` but not for other libraries. It's ok to have it here, in case some day `main` gets a third-party directory, but otherwise each library must set its own.

Reply:
I removed the change in the eslint file